### ISSUE TITLE
Animate the start up

### ISSUE THREAD

**Issue by ghost** [media-explorer/media-explorer#199]
Could we animate the start up of mex? such as grow the stage to full screen, it might be less visually jarring when launched

**Comment by semiot** [comment]
Yes! That would be great and would make for a nice cinematic start to the experience. When you start MEX, can you have the window launch in the centre of the screen then grow out to fill the screen?

**Comment by ghost** [comment]
Turns out that's quite difficult to achieve within the application it's self especially as it's being constructed. Though It has identified some issues with telling the window manager when the application is ready to be shown on screen, this is at an MX level and I'll file a bug upstream.

One thing we could look into is playing a start up video and then fade the ui over the top.
